decidim-core/app/views/layouts/decidim/header/_menu_form_search_mobile.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_tag(decidim.search_path, method: :get, id: "form-search-mobile", class: "filter-search") do %>
```

Comment:
```suggestion
<%= form_tag(decidim.search_path, method: :get, id: "form-<%= id %>", class: "filter-search") do %>
```
This id is what you get by specifing the `locals: {}` in the render

Reply:
oh ok. Thanks